pkg/pipeline/transform/transform_network.go

```diff
@@ -184,6 +185,40 @@ func fillInK8s(outputEntry config.GenericMap, rule api.NetworkTransformRule) {
 	}
 }
 
+const nodeZoneLabelName = "topology.kubernetes.io/zone"
+
+func fillInK8sZone(outputEntry config.GenericMap, rule api.NetworkTransformRule, kubeInfo kubernetes.Info) {
+	if rule.Kubernetes == nil || !rule.Kubernetes.AddZone {
+		//Nothing to do
+		return
+	}
+	switch kubeInfo.Type {
+	case kubernetes.TypeNode:
+		zone, ok := kubeInfo.Labels[nodeZoneLabelName]
+		if ok {
+			outputEntry[rule.Output+"_Zone"] = zone
```

Comment:
Since this is node related, I would suggest `HostZone` for consistency with `HostIP` & `HostName`

```suggestion
			outputEntry[rule.Output+"_HostZone"] = zone
```

Reply:
IMO this is not host related the same way. HostIP and HostName belongs to the host, the pod doest not have this IP or name.

But the container is in the zone, we juste get the information through the node.